lib/cadet/assessments/assessments.ex

```diff
@@ -139,6 +140,12 @@ defmodule Cadet.Assessments do
         user = %User{role: role}
       ) do
     if Timex.after?(Timex.now(), assessment.open_at) or role in @open_all_assessment_roles do
+      # ChatKit - create chatrooms
+      Submission
+      |> where(assessment_id: ^id)
+      |> Repo.all()
+      |> Enum.each(fn submission -> Room.create_rooms(submission) end)
```

Comment:
Let's simplify this. There should only be one submission, so we should do

```elixir
 Submission
      |> where(assessment_id: ^id)
      |> Repo.one()
      |> &Room.create_rooms/1
```

Reply:
Moved the room creation to answer_question/3. I think it makes more sense to have it after an answer is inserted or updated successfully.